ui/src/routes/components/Accounts/Accounts.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { Notifications } from '@components/Notifications';
+import { ConfirmEmail, Login } from '@pages/accounts';
+import { AccountsRoutes } from '@routes/urls';
+
+export class Accounts extends Component {
```

Comment:
How and where is this used, it is related to the `config.py` url variables?

Reply:
No this is just ui route to gather alltogether the operations regarding accounts (for now login, confirm email and in the future sign up, settings probably...).